src/server/pingserver/CMakeLists.txt

```diff
@@ -19,5 +19,10 @@ set(LIBS
     ${CMAKE_THREAD_LIBS_INIT})
 
 set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${CMAKE_HOME_DIRECTORY}/_bin)
-add_executable(${PROJECT_NAME}_pingserver ${SOURCE})
-target_link_libraries(${PROJECT_NAME}_pingserver ${MODULES} ${LIBS})
+set(TARGET_NAME ${PROJECT_NAME}_pingserver)
+
+add_executable(${TARGET_NAME} ${SOURCE})
+target_link_libraries(${TARGET_NAME} ${MODULES} ${LIBS})
+
+install(TARGETS ${TARGET_NAME} RUNTIME DESTINATION bin)
```

Comment:
That's my take – I assume all executable should be installed unless there's a reason not to, including (in the future) echoserver.

Thoughts? Concerns?